zebrad/tests/acceptance.rs

```diff
@@ -35,9 +35,9 @@ pub fn get_child(args: &[&str], tempdir: &PathBuf) -> Result<TestChild> {
     let mut cmd = test_cmd(env!("CARGO_BIN_EXE_zebrad"), &tempdir)?;
 
     Ok(cmd
+        .arg("-c")
+        .arg(tempdir.join("zebrad.toml"))
```

Comment:
Let's pass `ConfigMode` to `get_child`, then add a config argument if `config_mode != NoConfig`.

See the similar code here:
https://github.com/ZcashFoundation/zebra/pull/939/files#diff-33c347fc0c4532e93ecd68f7cef1fb07R25